tests/Kernel/MediaRenderTest.php

```diff
@@ -76,27 +76,26 @@ public function testDocumentMedia(): void {
 
     $build = $this->mediaViewBuilder->view($media, 'default');
     $html = $this->renderRoot($build);
-
     $crawler = new Crawler($html);
 
     // File wrapper.
     $file_wrapper = $crawler->filter('.ecl-file');
     $this->assertCount(1, $file_wrapper);
 
     // File row.
-    $file_row = $crawler->filter('.ecl-file .ecl-row');
+    $file_row = $crawler->filter('.ecl-file .ecl-file__container');
     $this->assertCount(1, $file_row);
 
     $file_title = $file_row->filter('.ecl-file__title');
     $this->assertContains('test.pdf', $file_title->text());
 
-    $file_info_language = $file_row->filter('.ecl-file__info span.ecl-file__language');
+    $file_info_language = $file_row->filter('.ecl-file__info div.ecl-file__language');
     $this->assertContains('English', $file_info_language->text());
 
-    $file_info_properties = $file_row->filter('.ecl-file__info span.ecl-file__properties');
+    $file_info_properties = $file_row->filter('.ecl-file__info div.ecl-file__meta');
     $this->assertContains('KB - PDF)', $file_info_properties->text());
 
-    $file_download_link = $file_row->filter('.ecl-file__download');
+    $file_download_link = $file_row->filter('a.ecl-file__download');
```

Comment:
This is out of scope, we can leave it but we need to make sure it does not impact any other issue.

Reply:
This media is already implemented with the new ECL2, so normally no impact.
I prefer to remove it because it's not consistent with the others.